src/components/common/ColonyActions/ActionDetailsPage/DefaultMotion/MotionPhaseWidget/StakingWidget/StakingInput/StakingSlider.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { useStakingSlider } from '~hooks';
+
+import {
+  StakingSliderDescription,
+  StakingSliderLabel,
+  StakingWidgetSlider,
+} from '.';
+
+const displayName =
+  'common.ColonyActions.DefaultMotion.StakingWidget.StakingSlider';
+
+interface StakingSliderProps {
+  isObjection: boolean;
+  canBeStaked: boolean;
+}
+
+const StakingSlider = ({ isObjection, canBeStaked }: StakingSliderProps) => {
```

Comment:
Refactored this so it can be used in the objection dialog without use of `useStakingWidgetContext` (since the provider doesn't wrap the dialog provider).